CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# 79.1
+- Update JxBrowser to `v7.38.2` (#7413)
+- "Open Android Module in Android Studio" action removed (#7103)
+- Fix for deprecation of `ActionUpdateThread.OLD_EDT` (#7330)
+- Deprecation of `ServiceExtensions.setPubRootDirectories` (#7142)
+- Fix Deep links tool window "hide" button does not work (#7321)
```

Comment:
Can we leave this out? We thought this was fixed by another change but was not actively addressed in this release.

Reply:
Removed.